### DISCUSSION TITLE
IIS filebeat module and default dashboards

### DISCUSSION THREAD

**Discussion by jjvquest** [Security-Onion-Solutions/securityonion#5518] [Unsupported Versions] [answered]
Is it possible, now or in the near future, to use Elastic dashboards that come with the Filebeat modules? For example the [IIS module](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-iis.html)?

I am currently sending the IIS logs with Filebeat (IIS module enabled) to the manager-search node (Logstash). From my understanding there is no need to enable the IIS Filebeat module on the manager-search, because there are no IIS logs there. 

Elastic has a [Filebeat IIS dashboard](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-iis.html#_example_dashboard_10). Is there some way to import/adjust?

Also: is my current setup the best in this case? I initially had some problems because my Filebeat version (7.14.1) was newer than the Security Onion version (7.13.4). Logstash logs showed:
`"pipeline with id [filebeat-7.14.1-iis-error-pipeline] does not exist"`
After installing the same Filebeat version on the webserver, logs made it into Elastic.

Jay - SO 2.3.70

**Comment by dougburks** [comment, marked as answer]
I've updated https://docs.securityonion.net/en/2.3/beats.html#version with the following:
> When downloading a Beats agent, make sure the version number matches the version of Elastic running on your Security Onion deployment.

A manager node requires a separate search node to store logs.  A manager-search node is able to store logs locally without a separate search node.  For more information, please see:
https://docs.securityonion.net/en/2.3/architecture.html#node-types

I haven't tried it, but you may be able to take an Elastic dashboard from a Filebeat module and manually import it.
